packages/flx/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ homepage: https://github.com/flutter/flutter/tree/master/packages/flx
 dependencies:
   bignum: ^0.0.7
   asn1lib: ^0.4.1
-  cipher: ^0.7.1
+  pointycastle: ^0.10.0-dev
```

Comment:
I see them published here: https://pub.dartlang.org/packages/pointycastle, so I think what you're hitting it something in another constraint forcing an older version. `pub` has a mode you can run it in that will give you more info about why it is or isn't selecting a version. Might be worth doing that to diagnose -